src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionCommentFilterTest.java

```diff
@@ -406,8 +396,7 @@ public void testInvalidMessageFormat() throws Exception {
         filterConfig.addProperty("messageFormat", "e[l");
 
         try {
-            final String[] suppressed = CommonUtil.EMPTY_STRING_ARRAY;
-            verifySuppressed(filterConfig, "InputSuppressionCommentFilter11.java", suppressed);
+            verifySuppressed(filterConfig, "InputSuppressionCommentFilter11.java");
```

Comment:
Method should be "execute" as we do not validate anything, we just run and expect exception

Reply:
`execute `method is failing and the error is `EXCEPTION IS EXPECTED`

Reply:
I have used execute method in the verifySuppressed method which takes 2 parameters.

Reply:
Do not use it inside, use it here.